src/OpenTelemetry.Instrumentation.EntityFrameworkCore/Implementation/EntityFrameworkInstrumentationEventSource.cs

```diff
@@ -59,6 +60,24 @@ public void InvalidPayload(string handlerName, string eventName)
         this.WriteEvent(4, handlerName, eventName);
     }
 
+    [NonEvent]
```

Comment:
small nit: we normally organize `NonEvent`s above `Event`s in the file.

Reply:
Corrected